apps/prairielearn/python/prairielearn.py

```diff
@@ -1304,8 +1302,9 @@ def string_to_2darray(s, allow_complex=True):
                 )
 
             # Iterate over columns
-            for j in range(0, n):
-                try:
+            j = 0
+            try:
+                for j in range(n):
```

Comment:
Why the switch from `range(0, n)`? I don't see a lint error when that's reverted. If that's a manual preference, consider applying it consistently everywhere.

Reply:
This is a manual preference; It was just something I cleaned up when I was touching this code.

Reply:
Should I apply that in this PR, or another one?

Reply:
Let's move those changes to another PR.

Reply:
Resolved in #11135